test/api/capi/test_capi_logging.cpp

```diff
@@ -0,0 +1,60 @@
+#include "capi_tester.hpp"
+#include "util/logging.h"
+
+using namespace duckdb;
+using namespace std;
+
+unordered_set<string> LOG_STORE;
+
+void WriteLogEntry(duckdb_timestamp timestamp, const char *level, const char *log_type, const char *log_message) {
+	LOG_STORE.insert(log_message);
```

Comment:
There are some open questions about thread safety and loggers, see https://github.com/duckdb/duckdb/blob/f484208c27f281ae0e6ae63104d18ff8d1e91796/src/include/duckdb/logging/log_storage.hpp#L148

This seems fine for now though, just something to keep in mind